### PR TITLE
Use websockets for progress indicator when updating practitioner contact info

### PR DESCRIPTION
**Pull request by rachaelparris**

### CODE REVIEW

**Review by codyseibert — APPROVED** [review]

**Comment by codyseibert** [comment]
This should be cool!

File: web-client/src/presenter/actions/updateUserContactInformationAction.js [hunk 1]
```diff
@@ -1,53 +1,25 @@
...
 
 /**
- * updates primary contact information
+ * updates user contact information (for practitioners)
  *
  * @param {object} providers the providers object
  * @param {object} providers.applicationContext the application context used for getting the getUser use case
  * @param {Function} providers.get the cerebral get function
- * @param {object} providers.path the next path in the sequence to call
  * @returns {object} alertSuccess
```
Comment (kkoskelin) [inline web-client/src/presenter/actions/updateUserContactInformationAction.js:9]: not any more.

Comment (rachaelparris) [reply web-client/src/presenter/actions/updateUserContactInformationAction.js:9]: I'll throw that in my next PR :)

File: web-client/src/views/UserContactEditProgress.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  {
+    userContactEditProgressHelper: state.userContactEditProgressHelper,
+  },
+  function UserContactEditProgress({ userContactEditProgressHelper }) {
+    return (
+      <div>
+        <div className="sticky-footer sticky-footer--space" />
+        <div className="sticky-footer sticky-footer--container">
+          <div className="usa-section grid-container padding-bottom-0 margin-top-1">
+            <div aria-live="polite" className="progress-user-contact-edit">
```
Comment (kkoskelin) [inline web-client/src/views/UserContactEditProgress.jsx:15]: This appears to be super-accessible.

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/views/UserContactEditProgress.jsx (hunk 2 above)
Comment (rachaelparris) [reply web-client/src/views/UserContactEditProgress.jsx:15]: I can't take credit! I copy-pasta'd straight from `BatchDownloadProgress.jsx` 😄

**Pull request merged**